fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -3126,7 +3126,8 @@ public PlanFragment visitPhysicalTableFunctionTableScan(OptExpression optExpress
 
             prepareContextSlots(node, context, tupleDesc);
 
-            scanNode.setLoadInfo(-1, -1, table, new BrokerDesc(table.getProperties()), fileGroups, false, 1);
+            int dop = ConnectContext.get().getSessionVariable().getSinkDegreeOfParallelism();
```

Comment:
`pipeline_sink_dop` is to send data when shuffle. You may need `pipeline_dop`

Reply:
The `pipeline_sink_dop` is used for load, while `pipeline_dop` is used for query. Users often need bigger dop for query.

Reply:
not yet, pipeline_sink_dop replaces the role of pipeline_dop in ingestion, because the default concurrency in ingestion is inconsistent with the query requirements